core/src/test/scala/integration/kafka/api/SslAdminIntegrationTest.scala

```diff
@@ -259,4 +275,22 @@ class SslAdminIntegrationTest extends SaslSslAdminIntegrationTest {
     assertTrue(metrics.nonEmpty, s"Unable to find metric $name: allMetrics: ${allMetrics.keySet.map(_.getMBeanName)}")
     metrics.map(_.asInstanceOf[Gauge[Int]].value).sum
   }
+
+  override def createAdminClient: Admin = {
```

Comment:
This was necessary because admin client is created with different security configurations for SSL tests than SASL and Plain. We could remove this and override the createConfig() method instead but the override is needed.

Reply:
I'm still not sure I understand. The `createConfig()` method is in `BaseAdminIntegrationTest` so it should create the right configuration for the running test automatically. Why is it able to correctly create the configuration when running with SASL_SSL but not with SSL?

Reply:
So I initially thought the `createConfig()` in BaseAdminIntegrationTest is creating different security properties by calling `TestUtils.adminClientSecurityConfigs()` than the security properties created as part of the set up in IntegrationTestHarness. However following all the properties `adminClientSecurityConfigs()` set, they are exactly the same except the `certAlias`. So basically BaseAdminIntegrationTest mostly just setting security properties twice for no good reason. This seems to upset SSL admin test, because of the mismatch in the certAlias. 

Removing the unnecessary call to `adminClientSecurityConfigs()` in BaseAdminIntegrationTest worked for all of the tests.